src/app/models/student.model.ts

```diff
@@ -8,7 +8,8 @@ export default class Student {
   firstname: string;
   lastname: string;
   gender: Gender;
-  class: Class;
+  class?: Class;
+  class_id: string;
```

Comment:
Hi @eran1205,
good question.
The DB users table has class_id field and so does the user interface,
but the student interface is different for some reason.
I fixes this but since it's not clear was meant by "class" field, I let it be to prevent regression